src/services/arweave/WarpDataProvider.ts

```diff
@@ -149,4 +158,87 @@ export class WarpDataProvider implements SmartweaveDataProvider {
       return error;
     }
   }
+
+  async handleTransactionExecution({
+    contractType,
+    interactionType,
+    transactionData,
+  }: {
+    contractType: ContractType;
+    interactionType: AntInteraction | RegistryInteraction;
+    transactionData: any;
+  }): Promise<string | undefined> {
```

Comment:
i'd expect every interaction to return the `originalTxId` - or an error thrown

Reply:
changed